tests/baselines/reference/user/bcryptjs.log

```diff
@@ -8,7 +8,7 @@ node_modules/bcryptjs/src/bcrypt.js(150,5): error TS2322: Type 'string | undefin
   Type 'undefined' is not assignable to type 'string'.
 node_modules/bcryptjs/src/bcrypt.js(160,14): error TS2366: Function lacks ending return statement and return type does not include 'undefined'.
 node_modules/bcryptjs/src/bcrypt.js(238,14): error TS2366: Function lacks ending return statement and return type does not include 'undefined'.
-node_modules/bcryptjs/src/bcrypt/impl.js(516,22): error TS2345: Argument of type 'Int32List' is not assignable to parameter of type 'number[]'.
+node_modules/bcryptjs/src/bcrypt/impl.js(516,22): error TS2345: Argument of type 'number[] | Int32Array' is not assignable to parameter of type 'number[]'.
```

Comment:
Interesting - we _lost_ an alias here. @sandersn do you know if the `type Int32List = Int32Array | GLint[];` alias was removed from `dom` (and inlined at usages)? That's probably a breaking change.